lib/states/wi/madison.R

```diff
@@ -8,86 +8,134 @@ source("common.R")
 # each year, this seems reasonable.
 load_raw <- function(raw_data_dir, n_max) {
   # NOTE: "IBM" is the officers department ID
-  cit <- load_single_file(
+  cit_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_citations.csv",
     n_max
   )
-  warn <- load_single_file(
+  warn_old <- load_single_file(
     raw_data_dir,
     "mpd_traffic_stop_request_warnings.csv",
     n_max
   )
+  
+  cit_new <- load_single_file(
+    raw_data_dir,
+    "elcis_with_demographics.csv",
+    n_max
+  )
+  
+  warn_new <- load_single_file(
+    raw_data_dir,
+    "warnings_with_demographics.csv",
+    n_max
+  )
+  
+  ## NOTE: The new data starts in Jan. 2018 
+  ## and the old data covers through Sept. 2017, 
+  ## so we're still missing Oct.-Dec. 2017
+  new_d <- bind_rows(
+    cit_new$data,
+    warn_new$data
+  )
+  
   bundle_raw(
-    bind_rows(cit$data, warn$data),
-    c(cit$cloading_problems, warn$loading_problems)
+    bind_rows(cit_old$data, warn_old$data, new_d),
+    c(cit_old$loading_problems,
+      warn_old$loading_problems,
+      cit_new$loading_problems,
+      warn_new$loading_problems)
```

Comment:
This is just a personal preference, but I find it slightly easier to read when it's written as
```R
c(
  cit_old$loading_problems,
  warn_old$loading_problems,
  cit_new$loading_problems,
  warn_new$loading_problems
)
```